### PR TITLE
Fix bug in is_sparse() method, enable sparse test in training

### PR DESCRIPTION
**Pull request by kalyc**
### Summary
Fix bug in is_sparse() method, enable sparse test in training
Disable sparse concat test for testing CI build

### Related Issues
Examples were failing locally - verified that they work locally now
Unit tests were also failing locally under test_training - verified that they pass now

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n]

### CODE REVIEW

**Comment by roywei** [comment]
closing and reopen to trigger CI

File: keras/backend/mxnet_backend.py [hunk 1]
```diff
@@ -172,9 +172,10 @@ def is_sparse(tensor):
     if hasattr(tensor, 'tocoo'):
         return True
     elif isinstance(tensor, KerasSymbol):
-        if isinstance(_forward_pass(tensor)[0], mx.ndarray.sparse.CSRNDArray) or \
-                isinstance(_forward_pass(tensor)[0], mx.ndarray.sparse.RowSparseNDArray):
-                return True
+        if len(tensor.get_bind_values()) > 0:
+            if isinstance(_forward_pass(tensor)[0], mx.ndarray.sparse.CSRNDArray) or \
```
Comment (sandeep-krishnamurthy) [inline keras/backend/mxnet_backend.py]: saving _forward_pass(tensor) will save extra forward pass.

Comment (kalyc) [reply keras/backend/mxnet_backend.py]: added

**Comment by kalyc** [comment]
Related issue - to enable sparse concat test - https://github.com/awslabs/keras-apache-mxnet/issues/179

**Pull request merged**